components/builder-worker/src/server.rs

```diff
@@ -39,6 +39,9 @@ use crate::log_forwarder::LogForwarder;
 use crate::network::NetworkNamespace;
 use crate::runner::{studio, RunnerCli, RunnerMgr};
 
+/// Interval for main thread to check cancel status
+pub const BUILD_CANCEL_WAIT_SECS: u64 = 15;
```

Comment:
I'm.. very confused as to what's going on here. It looks like my indexes might be boogered up

Reply:
rustfmt issue?